packages/mephisto-task/README.md

```diff
@@ -166,16 +166,18 @@ function MyApp() {
         destroy,
         sendMessage
 
-        agentState,
         agentStatus,
 
         connectionStatus,
     } = useMephistoLiveTask(
         onConnectionStatusChange: (connectionStatus) => {
```

Comment:
Just noticed a typo here, these props should be passed wrapped in an object, this is invalid javascript.

Fix:

```js
    } = useMephistoLiveTask({   // <-- notice the { here to start the object definition
        onConnectionStatusChange: (connectionStatus) => {
```